cmd/version/options.go

```diff
@@ -34,6 +34,7 @@ func NewRosaVersionOptions() (*RosaVersionOptions, error) {
 	return &RosaVersionOptions{
```

Comment:
@ciaranRoche Could we add a quick unit test for the `NewRosaVersionOptions` function just so we don't regress this at any point in the future 👍